specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,8278 @@
+            "AzureDataExplorer",
+            "AzureFileService",
+            "AzureSqlDatabase",
+            "AmazonPostgreSql",
+            "AzurePostgreSql",
+            "SqlServerDatabase",
+            "AzureSqlDatabaseManagedInstance",
+            "AzureSqlDataWarehouse",
+            "AzureMySql",
+            "AzureStorage",
+            "Teradata",
+            "Oracle",
+            "SapS4Hana",
+            "SapEcc",
+            "PowerBI"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "DataSourceType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Scan": {
+      "required": [
+        "kind"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "enum": [
+            "AzureSubscriptionCredential",
+            "AzureSubscriptionMsi",
+            "AzureResourceGroupCredential",
+            "AzureResourceGroupMsi",
+            "AzureSynapseWorkspaceCredential",
+            "AzureSynapseWorkspaceMsi",
+            "AzureSynapseCredential",
+            "AzureSynapseMsi",
+            "AdlsGen1Credential",
+            "AdlsGen1Msi",
+            "AdlsGen2Credential",
+            "AdlsGen2Msi",
+            "AmazonAccountCredential",
+            "AmazonS3Credential",
+            "AmazonS3RoleARN",
+            "AmazonSqlCredential",
+            "AzureCosmosDbCredential",
+            "AzureDataExplorerCredential",
+            "AzureDataExplorerMsi",
+            "AzureFileServiceCredential",
+            "AzureSqlDatabaseCredential",
+            "AzureSqlDatabaseMsi",
+            "AmazonPostgreSqlCredential",
+            "AzurePostgreSqlCredential",
+            "SqlServerDatabaseCredential",
+            "AzureSqlDatabaseManagedInstanceCredential",
+            "AzureSqlDatabaseManagedInstanceMsi",
+            "AzureSqlDataWarehouseCredential",
+            "AzureSqlDataWarehouseMsi",
+            "AzureMySqlCredential",
+            "AzureStorageCredential",
+            "AzureStorageMsi",
+            "TeradataTeradataCredential",
+            "TeradataTeradataUserPass",
+            "TeradataUserPass",
+            "OracleOracleCredential",
+            "OracleOracleUserPass",
+            "SapS4HanaSapS4HanaCredential",
+            "SapS4HanaSapS4HanaUserPass",
+            "SapEccSapEccCredential",
+            "SapEccSapEccUserPass",
+            "PowerBIDelegated",
+            "PowerBIMsi"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ScanAuthorizationType",
+            "modelAsString": true
+          }
+        },
+        "scanResults": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScanResult"
+          },
+          "readOnly": true
+        }
+      },
```

Comment:
most of the models/properties miss description.

Reply:
added description on properties field